### PR TITLE
Update purefb_s3user with access policy option at user creation

### PR DESCRIPTION
**Pull request by sdodsley**
##### SUMMARY
Purity 3.2 adds object-store user access policies.
This patch enables policies to be assigned to a user at creation.
If no policy is provided then the full-access policy is applied
If a policy or list of policies is supplied, not including full-access, then ensure full-access is not granted.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
purefb_s3user.py

### CODE REVIEW

File: collections/ansible_collections/purestorage/flashblade/plugins/modules/purefb_s3user.py [hunk 1]
```diff
@@ -63,6 +63,14 @@
     - Access key secret for access key to import
     type: str
     version_added: "1.4.0"
+  policy:
+    description:
+    - User Access Policies to be assigned to user on creation
+    - To amend policies use the I(purefb_userpolicy) module
+    - If not specified, I(pure\:policy/full-access) will be added
```
Comment (rdeenik) [inline collections/ansible_collections/purestorage/flashblade/plugins/modules/purefb_s3user.py:70]: Remove line 70 and add following:

```yaml
    default: policy/full-access
```

**Review by rdeenik — APPROVED** [review]
Small nit for docs, otherwise lgtm!

File: collections/ansible_collections/purestorage/flashblade/plugins/modules/purefb_s3user.py (hunk 1 above)
Comment (sdodsley) [reply collections/ansible_collections/purestorage/flashblade/plugins/modules/purefb_s3user.py:70]: Actually, the APIs will create that default policy for a user without needing to set it - hence why I need to actually remove it if not specified in the `policy` parmaeter

Comment (rdeenik) [reply collections/ansible_collections/purestorage/flashblade/plugins/modules/purefb_s3user.py:70]: That makes sense then.

**Pull request merged**